### DISCUSSION TITLE
When import spatie backup file on mysql it give a error

### DISCUSSION THREAD

**Discussion by Amits09** [spatie/laravel-backup#1704] [Q&A]
This is the error
#1062 - Duplicate entry '0' for key 'PRIMARY'

**Comment by angeljqv** [comment]
Means that '0' key already exists, maybe `autoincrement` problem

**Comment by gmmx** [comment]
I'm having this issue recently

### Steps to Reproduce
-  destination disk : S3
php artisan backup:run --only-db


### Expected Behavior
All values should be correctly aligned with the id at first, ending by created_at . The values to insert seem compact / minified in one line.

```

INSERT INTO `activity_log` VALUES (1,'default','Access Leads',NULL,NULL,NULL,'App\\Models\\User',1,'[]',NULL,'2022-12-27 23:26:17','2022-12-27 23:26:17'),(2,'default','Access Leads',NULL,NULL,NULL,'App\\Models\\User',1,'[]',NULL,'2022-12-28 00:27:30','2022-12-28 00:27:30'),(3,'default','Access Users',NULL,NULL,NULL,'App\\Models\\User',7,'[]',NULL,

```

### Actual Behavior
The date values are being placed randomly. The file size is bigger because there are a lot of lines
Getting the error : "ERROR 1062 (23000) at line 91131: Duplicate entry '18' for key ..."
```

INSERT INTO `activity_log` VALUES
(NULL,1,'App\\Models\\User','2022-12-27 23:26:17','Access Leads',NULL,1,'default','[]',NULL,NULL,'2022-12-27 23:26:17'),
(NULL,1,'App\\Models\\User','2022-12-28 00:27:30','Access Leads',NULL,2,'default','[]',NULL,NULL,'2022-12-28 00:27:30'),
(NULL,7,'App\\Models\\User','2022-12-28 08:29:50','Access Users',NULL,3,'default','[]',NULL,NULL,'2022-12-28 08:29:50'),

```


### Environment
Spatie-backups version: 1.3.1
Laravel version: 9
PHP version: 8.2
MySQL version: 8.0

**Reply by angeljqv** [reply]
>**Steps to Reproduce**
>php artisan backup:run --only-db

The steps to reproduce work correctly

**Reply by gmmx** [reply]
Steps to Reproduce
dump with plugin:
- php artisan backup:run --only-db
dump with mysqldump 
- mysqldump [...]  > backup.sql

the two files backup.sql and the unzipped SQL file are different.

**Reply by angeljqv** [reply]
everything works as expected on my side

![image](https://github.com/spatie/laravel-backup/assets/79208641/bbda5858-ae20-43c7-aa91-6d636259e955)
I see you just modified and added `s3`, i can't check s3, sorry

**Reply by labomatik** [reply]
Got the same issue with a S3 backup since few weeks now (it was working perfectly before without any changes in the config)
A mysqldump in a console is working fine...

**Reply by gmmx** [reply]
Adding --complete-insert did the trick

config/database.php
```
 'dump' => [
                'add_extra_option' => '--complete-insert',
            ],
```

**Reply by A1exMiranda** [reply]
Did you guys found anything? Backups were working as they should but suddenly they were getting this random order.

PS add_extra_option didn't work.

Thanks in advance!
